modules/core/src/parallel.cpp

```diff
@@ -966,8 +981,11 @@ int getNumberOfCPUs_()
     static unsigned ncpus_impl_cpuset = (unsigned)getNumberOfCPUsImpl("/sys/fs/cgroup/cpuset/cpuset.cpus");
```

Comment:
What about cpuset? Apparently on my system with cgroups2 (Ubuntu 22.04 - based) it has changed location to `/sys/fs/cgroup/cpuset.cpus` or `cpuset/cpus.effective` (idk which is right).

```
$ docker run --cpuset-cpus 1-3,8 -it ubuntu:22.04
root@f0e45a761cea:/# cat /sys/fs/cgroup/cpuset.cpus
1-3,8
```

Reply:
It is a workaround for callers.

Because we could assign CPUs and also limit their usage:

```
docker run --rm --cpuset-cpus 1-3,8 --cpu-quota=300000 --cpu-period=100000 -it ubuntu:22.04
root@d4db947e6eba:/# cat /sys/fs/cgroup/cpuset.cpus
1-3,8
root@d4db947e6eba:/# cat /sys/fs/cgroup/cpu.max    
300000 100000
```